### DISCUSSION TITLE
Should Suzie's Comfyroll Studio extension be replaced by a fork?

### DISCUSSION THREAD

**Discussion by andreszs** [Comfy-Org/ComfyUI#6123] [General]
The original [Comfyroll Studio](https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes) developer has vanished almost a year ago, not only from Github but from all networks.

Considering this is an extremely popular extension, should **ComfyUI Manager** step in and replace it by a forked version? I believe this has been done with other extensions, but I'm not sure how long should we wait to take that action. In the context of ComfyUI and SD, a full year is way too much to wait for anyone.

Note that I'm not a fan of that extension, actually I've barely used its nodes but surely many other users rely on that extension for their workflows.

**Comment by ltdrdata** [comment]
Some custom nodes function well even if they are not maintained, so replacements are not made solely based on the duration of maintenance inactivity.

1. When the original author explicitly declares that they will no longer maintain the node.
2. When it is evident that the node has been effectively abandoned, such as when a pull request has been submitted and efforts to contact the original author have been made, but there has been no response for a long period (at least one month or more).

For these two cases, if there is a user request, the existing node will be moved to a legacy channel, and the replacement process will proceed.
